src/rofi_rbw/rofi_rbw.py

```diff
@@ -11,7 +11,7 @@
     from rofi_rbw.action import Action
     from rofi_rbw.clipboarder import Clipboarder
     from rofi_rbw.typer import Typer
-    from rofi_rbw.selector import Selector
+    from rofi_rbw.selector import Selector, SelectorResponse
```

Comment:
Could you please add it to the `except` branch too?

Reply:
it's already in the except branch

Reply:
I'm sorry, I missed it :(